internal/dev_server/ui/src/main.tsx

```diff
@@ -28,6 +29,7 @@ const Root = () => {
     <React.StrictMode>
       <IconProvider>
         <App />
```

Comment:
Should it be at the top? That's where most of the latent interactions happen.

Reply:
TBH, not sure.  I think just having it somewhere relatively global is standard.

Don't think it matters that much here?

Reply:
wym by at the top?

Reply:
oh like at the top of the page. Screenshot makes it look like maybe it's at the bottom of the page?

Reply:
Location in the markup seems fine.

Reply:
ohhh, yea... bottom of the page is standard for toasts and I don't think We can change that.  The <ToastProvider> doesn't actually control that either.  The position is absolute no matter where we render the provider

Reply:
When using it it's clear enough to give some confirmation 👍🏻

Reply:
🆒